scripts/test_repository.py

```diff
@@ -49,6 +45,7 @@
 import os
 import sys
 import time
+import shlex
```

Comment:
I use it in line 123. It may not be strictly neccesary there but its usually the better option to bare split() calls if you are handling shell commands as far as I know. Is there a special reason you want to get rid of it?

Reply:
Since `shlex` is not part of https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml I would drop it.

Reply:
But it is standard library so it should not be required to be in rosdep
see https://docs.python.org/3.9/library/index.html